src/instance/config.go

```diff
@@ -11,31 +11,63 @@ import (
 )
 
 const (
-	cfg_folder     = "cfg"
 	server_ini     = "server_cfg.ini"
 	entry_list_ini = "entry_list.ini"
 	sep            = "\n"
 )
 
-func writeConfig(config *model.Configuration) error {
-	iniPath := filepath.Join(cfg_folder, int64ToStr(config.Id))
-	if err := os.MkdirAll(iniPath, 0755); err != nil {
-		log.WithFields(log.Fields{"err": err}).Error("Error creating cfg folder")
+func GetServerCfgFileName() string {
+	return server_ini
+}
+
+func GetEntryListFileName() string {
+	return entry_list_ini
+}
+
+func GetConfigPath(config *model.Configuration) string {
+	configPath := filepath.Join(os.Getenv("ACWEB_CONFIG_DIR"), int64ToStr(config.Id))
+	return configPath
+}
+
+func GetServerCfgPath(config *model.Configuration) string {
+	iniFile := filepath.Join(GetConfigPath(config), server_ini)
+	return iniFile
+}
+
+func GetEntryListPath(config *model.Configuration) string {
+	iniFile := filepath.Join(GetConfigPath(config), entry_list_ini)
+	return iniFile
+}
+
+func writeIniFile(config *model.Configuration, ini, filename string) error {
+	if err := ioutil.WriteFile(filename, []byte(ini), 0775); err != nil {
+		log.WithFields(log.Fields{"err": err}).Errorf("Error writing %s", filename)
```

Comment:
Remove the `... %s", filename` and user Error(). Additional information should be added as fields: `log.WithFields(log.Fields{"err": err, "filename": filename}).Error(...`.